src/ScaffoldCommand.php

```diff
@@ -136,7 +136,9 @@ protected function getInput()
 
         $defaultSettings['repo_url'] = $defaultSettings['base_repo_url'] . $this->parameters['composer_name'];
         $this->askForSetting($defaultSettings, 'repo_url', 'Repo URL');
-        $this->parameters['repo_name'] = substr(strrchr($this->parameters['repo_url'], '/'), 1);
+        $repoParts = explode('/', $this->parameters['repo_url']);
```

Comment:
Looks like it would make more sense to do this against `$this->parameters['composer_name']` instead of `$this->parameters['repo_url']`?